client/ayon_core/lib/ayon_info.py

```diff
@@ -24,15 +30,41 @@ def is_running_from_build():
 
     Returns:
         bool: True if running from build.
-    """
 
+    """
     executable_path = os.environ["AYON_EXECUTABLE"]
     executable_filename = os.path.basename(executable_path)
     if "python" in executable_filename.lower():
         return False
     return True
 
 
+def is_using_ui_executable():
```

Comment:
```suggestion
def is_using_windows_ui_executable():
```

Should we make it clear that this is windows-specific from the function name as well?

Reply:
You're mind reader. We were just discussing it, and came to conclusion that `is_using_ayon_console` might be the best.

Reply:
Now I realized I also have to switch the logic...